addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/BodyGenerator.java

```diff
@@ -122,6 +129,22 @@ private String generateFromArraySchema(ArraySchema schema) {
         return createJsonArrayWith(generate(schema.getItems()));
     }
 
+    private String generateFromBinarySchema(BinarySchema schema, Boolean isImage) {
+        if (isImage) {
+
+            // Dummy data
+            byte[] data = new byte[] {110, -12, 34, -18, 11, 69, 20, 11, 51, 26, 27, 14};
+
+            String imageFileContents = new String(data);
+            return imageFileContents;
```

Comment:
- The value can be returned without the variable.
- Should use a `Charset` to create the string.
- Should be a constant, not necessary to create a new string each time (it can also be public to be used in the tests).